src/server/config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+    libaryPath: "/home/thom/dev/repos/tune-stream/music"

```

Comment:
909fe1b
I've added a local config, it's not tracked by git, but you can add variables to `config.local.js` and they will stay on your system. The app does not care if it isn't there